alerts/etcdNoLeader.md

```diff
@@ -0,0 +1,38 @@
+# etcdNoLeader
+
+## Meaning
+
+This alert is triggered when etcd cluster does not have a leader for more than 1
+minute.
+
+## Impact
+
+When there is no leader, Kubernetes and OpenShift APIs will not be able to work
+as expected and cluster cannot process any writes, and any write requests are
+queued for processing until a new leader is elected. Operations that preserve
+the health of the workloads cannot be performed.
+
+## Diagnosis
+
+Check the logs of etcd containers to see any further information and to verify
+that etcd does not have leader. Logs should contain something like `etcdserver:
+no leader`. etcd containers are running in the `openshift-etcd`
+namespace/project and in the `etcd` container.
+
+This can occur multiple control plane nodes are powered off or are unable to
+connect each other via the network. Check that all control plane nodes are
+powered and that network connections between each machine are functional.
```

Comment:
We could make this as the first point to diagnose. So sequencially it would be:
1. Check if all etcd instances/control plane nodes are up and powered on.
2. If all nodes are up check for etcd logs for events like `etcdserver: no leader` etc..

Reply:
I did it slightly different based on other existing runbooks, so to have the same style, what do you think of this?

Reply:
Understood. I think the point of having followup and improvisation in the end can be dealt with together. To start with, this seems fine.